src/main/webapp/style.css

```diff
@@ -11,13 +18,17 @@ html {
   position: relative;
 }
 
-#add-btn-cont {
+#add-btn-cont,
+#view-btn-cont {
+  align-items: center !important;
   margin: 30px auto auto auto;
```

Comment:
I think you forgot to alphabetize the declarations in this selector.

Reply:
fixed